### PR TITLE
Add ability to create API keys with 'register executor' capability.

### PR DESCRIPTION
**Pull request by vadimberezniker**
---

**Version bump**: Patch

### CODE REVIEW

File: enterprise/app/api_keys/api_keys.tsx
Comment (bduffany) [inline enterprise/app/api_keys/api_keys.tsx:426]: nit: I think this could be simplified to `return Boolean(apiKey?.capability?.some(...))`

**Review by bduffany — APPROVED** [review]

File: enterprise/app/api_keys/api_keys.tsx
Comment (bduffany) [inline enterprise/app/api_keys/api_keys.tsx:144]: nit: slightly more idiomatic to do `[...apiKey.capability]`

File: enterprise/app/api_keys/api_keys.tsx
Comment (vadimberezniker) [reply enterprise/app/api_keys/api_keys.tsx]: Done.

File: enterprise/app/api_keys/api_keys.tsx
Comment (vadimberezniker) [reply enterprise/app/api_keys/api_keys.tsx]: Done.

**Comment by vadimberezniker** [comment]
@siggisim I'll wait for your LGTM before merging

**Review by siggisim — APPROVED** [review]

**Pull request merged**
